src/store/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from "redux";
+import moviesReducer from "./moviesReducer";
+
+const allReducers = combineReducers({
```

Comment:
if you don't have nor plan to have more that 1 reducer, you don't need combineReducers

Reply:
Yes I know, wasn't sure about the project's future needs that's why